isaric/parsers/predicovid.toml

```diff
@@ -0,0 +1,477 @@
+    ]
+
+  [subject.age]
+    field = "age_estimateyears"
+
+  [subject.sex_at_birth]
+    field = "sex"
+    description = "sex at birth"
+    values = { 1 = "male", 2 = "female" }
+
+  [subject.ethnicity]
+    description = "Ethnic groups"
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [
+      { field = "ethnic___1", values = { 1 = "Arab" } },
+      { field = "ethnic___2", values = { 1 = "Black" } },
+      { field = "ethnic___3", values = { 1 = "East_Asian" } },
+      { field = "ethnic___4", values = { 1 = "South_Asian" } },
+      { field = "ethnic___5", values = { 1 = "West_Asian" } },
+      { field = "ethnic___6", values = { 1 = "Latin_American" } },
+      { field = "ethnic___7", values = { 1 = "White" } },
+      { field = "ethnic___8", values = { 1 = "Aboriginal_First_Nations" } },
+      { field = "ethnic___9", values = { 1 = "Other" } },
+      { field = "other_ethnic" },
+    ]
+
+  [subject.works_microbiology_lab]
+    field = "labwork_erterm"
+    ref = "Y/N/NK"
+
+  [subject.works_healthcare]
+    field = "healthwork_erterm"
+    ref = "Y/N/NK"
+
+  [subject.pregnancy]
+    field = "pregyn_rptestcd"
+    values = { 1 = true, 0 = false }
+
+  [subject.pregnancy_outcome]
+    field = "pregout_rptestcd"
+    values = { 1 = "live birth", 2 = "still birth" }
+
+  [subject.pregnancy_post_partum]
+    field = "postpart_rptestcd"
+    values = { 1 = true, 0 = false }
+
+  [subject.pregnancy_gestational_age_weeks]
+    field = "egestage_rptestcd"
+    description = "Gestational age (weeks amenorrhea)"
+
+  [subject.has_smoking]
+    combinedType = "firstNonNull"
+    fields = [
+      { field = "tabac", description = "Just before your diagnosis, did you smoke", values = { 1 = "yes", 2 = "yes", 3 = "never" } },
+      { field = "tabac_two", description = "In the past, have you smoked?", values = { 1 = "yes", 2 = "yes", 3 = "never" } },
+    ]
+
+
+  [subject.has_asthma]
+    combinedType = "any"
+    fields = [
+      { field = "asthme", description = "Have you ever had asthma?", values = { 1 = true, 0 = false } },
+    ]
+
+  [subject.has_chronic_kidney_disease]
+    combinedType = "any"
+    fields = [
+      { field = "antec_five", values = { 1 = true, 0 = false }, description = "chronic renal failure dialysis" },
+    ]
+
+
+  [subject.has_diabetes]
+    combinedType = "any"
+    fields = [
+      { field = "antec_one", values = { 1 = true, 0 = false }, description = "Diabetes (type 1 or 2)" },
+    ]
+
+  [subject.has_liver_disease]
+    combinedType = "any"
+    fields = [
+      { field = "antec_six", values = { 1 = true, 0 = false }, description = "chronic liver disease" },
+    ]
+
+  [subject.has_immunosuppression]
+    field = "antec_seven"
+    description = "a disease known to reeduce your immune defense"
+    values = { 1 = true, 0 = false }
+
+
+  # If subject is filling in a follow-up form at month 12, we assume they're not dead!
```

Comment:
Sounds reasonable. Is the field empty for some individuals? just curious ..

Reply:
It's a separate followup survey that was only done by some subjects, so yes there are lots of individuals where the 'questionnaire' field doesn't contain a monthly/month_12 entry